pkg/network/ebpf/c/tracer-bind.h

```diff
@@ -0,0 +1,136 @@
+#ifndef __TRACER_BIND_H
+#define __TRACER_BIND_H
+
+#ifndef COMPILE_CORE
+#include <linux/net.h>
+#include <linux/socket.h>
+#include <uapi/linux/in.h>
+#endif
+
+#include "bpf_endian.h"
+#include "bpf_telemetry.h"
+#include "bpf_helpers_custom.h"
+
+#include "tracer.h"
+#include "port.h"
+#include "sock.h"
+#include "tracer.h"
+#include "tracer-maps.h"
+
+//region sys_enter_bind
```

Comment:
Is this comment significant ?

Reply:
It doesn't seem like it; I am going to remove it.